### ISSUE TITLE
documentation page not find

### ISSUE THREAD

**Issue by TheRainInSpain** [heathsc/gemBS#102]
When I tried to read the documentation, I found that the 404 page not found error happened, it has lasted for about 2 weeks.
But I could read that two weeks ago. Is there some methods to read the documentation. Thanks.

**Comment by daisyko** [comment]
Yes. I also noticed the issue. I would leave a comment here to follow on the updates for this issue.

**Comment by quirinmanz** [comment]
Found this: https://github.com/heathsc/gemBS-rs/issues/15
